src/renderers/webgl/WebGLTextures.js

```diff
@@ -1273,7 +1333,54 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 		_gl.bindRenderbuffer( _gl.RENDERBUFFER, renderbuffer );
 
-		if ( renderTarget.depthBuffer && ! renderTarget.stencilBuffer ) {
+		if ( renderTarget.isWebGLMultiviewRenderTarget === true ) {
+
+			const useMultisample = useMultisampledRTT( renderTarget );
+			const numViews = renderTarget.numViews;
+
+			const depthTexture = renderTarget.depthTexture;
+			let glInternalFormat = _gl.DEPTH_COMPONENT24;
+			let glDepthAttachment = _gl.DEPTH_ATTACHMENT;
+
+			if ( depthTexture && depthTexture.isDepthTexture ) {
+
+				if ( depthTexture.type === FloatType ) {
+
+					glInternalFormat = _gl.DEPTH_COMPONENT32F;
+
+				} else if ( depthTexture.type === UnsignedIntType ) {
+
+					glInternalFormat = _gl.DEPTH_COMPONENT24;
```

Comment:
DeepScan complains here since `glInternalFormat` already has the value of `_gl.DEPTH_COMPONENT24`.

https://deepscan.io/dashboard/#view=project&pid=19901&iid=120730212&tid=16600&subview=pull-request&prid=1229149